ogr/abstract.py

```diff
@@ -157,14 +133,24 @@ def __init__(
     def __str__(self) -> str:
         comment = f"{self.comment[:10]}..." if self.comment is not None else "None"
         return (
-            f"PRComment("
+            f"Comment("
             f"comment='{comment}', "
             f"author='{self.author}', "
             f"created='{self.created}', "
             f"edited='{self.edited}')"
         )
 
 
+class IssueComment(Comment):
+    def __str__(self) -> str:
+        return "Issue" + super().__str__()
+
+
+class PRComment(Comment):
+    def __str__(self) -> str:
+        return "PR" + super().__str__()
```

Comment:
What about the backward link to the original pull-request object?

Reply:
You mean the object from package? Or the ones that are supposed to be implemented (ogr's Issue, PR)?

Reply:
The OGR object. i would go with this logic:

```python
pr = project.pr_create(...)
assert pr.project == project
pr_comment = pr.comment(...)
assert pr_comment.pull_request == pr
```

Reply:
Not sure if it's possible right now, since we're using only the objects from github/gitlab packages, pagure doesn't even touch PR in any way (except the comments)... You want to do it in this PR or later when implementing Issue/PR?

Reply:
> Not sure if it's possible right now

Yes, I am not sure as well. This is the ideal state I would like to achieve..;-)

Leave it for the next PR, since we would need to construct the `caller` after the real call.